### ISSUE TITLE
Question about denoiser training

### ISSUE THREAD

**Issue by cyk990422** [zkf1997/DART#23]
Regarding the training part of the MLD denoiser, I found my loss function to be quite strange. I didn't make any changes. Is this loss function correct?

![Image](https://github.com/user-attachments/assets/2c9118bb-1814-4471-b7bd-8bcf9ba301fa)

**Comment by zkf1997** [comment]
Hi,

During the scheduled training stage, the training loss can increase because rollout training, which uses generated history, poses a challenge for the model that was initially trained on ground truth history. Additionally, the proportion of rollout training gradually increases during this second stage, contributing to the rise in training loss.
